src/test/testUtil.ts

```diff
@@ -240,6 +240,47 @@ export async function assertTextEditorContains(contents: string): Promise<void |
     }
 }
 
+/**
+ * Create and open an editor with provided fileText, fileName and options. If folder is not provided,
+ * will create a temp worksapce folder which will be automatically deleted in testing environment
+ */
+export async function openATextEditorWithText(
```

Comment:
Now that this is lifted, we can remove the old one: https://github.com/aws/aws-toolkit-vscode/blob/bfd220fb09749ef345fb94c58fabb668ae9fa5b3/src/testInteg/stepFunctions/visualizeStateMachine.test.ts#L74